Problem1.java

```diff
@@ -0,0 +1,28 @@
+/**
+LeetCode Submitted : YES
+Space Complexity : O(Number of timestamp tuples)
+Time Complexity : O(NlogN) + O(NlogN)
```

Comment:
Please explain how you calculated the time complexity

Reply:
Time Complexity : 
O(NlogN) for sorting the timestamps based on first index of each tuple.
O(logN) if we cannot schedule any conference in parallel way so we will call heapify i.e. log(N) N times. So, complexity will be O(NlogN) in worst case.

Reply:
okay. Good.